python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -19,11 +19,12 @@
 module_logger = logging.getLogger(__name__)
 module_logger.setLevel(logging.INFO)
 
-
 try:
     import torch
     import torch.nn as nn
     import torchvision
+    from torchvision import transforms as T
+    from vision_explanation_methods.explanations import common as od_common
 except ImportError:
     module_logger.debug('Could not import torch, required if using a PyTorch model')
```

Comment:
nit: the error message here is "Could not import torch, required if using a PyTorch model", but this can be thrown with this PR if:
vision_explanation_methods package import failed
torchvision package import failed
so I think it might be better to move those into their own try/except sections?